### PR TITLE
Scraping example expenses

### PR DESCRIPTION
**Pull request by ragog**

### CODE REVIEW

File: blog/posts/scraping-example-expenses.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Scraping behind login
+subTitle: An example in accessing account-specific information
```
Comment (maxigimenez) [inline blog/posts/scraping-example-expenses.md:3]: 👌

File: blog/snippets/playwright/scraping-example-expenses.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    )
+
+    filteredPrices = filteredPrices.concat(
+      prices.filter((price) => price.includes(currency))
+    )
+    await page.waitForSelector('li.a-last')
+    await page.click('li.a-last')
+  }
+
+  const cleanPrices = filteredPrices.map(function (x) {
```
Comment (maxigimenez) [inline blog/snippets/playwright/scraping-example-expenses.js:53]: nit. use arrow function like the rest of the example

File: blog/snippets/puppeteer/scraping-example-expenses.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    )
+
+    filteredPrices = filteredPrices.concat(
+      prices.filter((price) => price.includes(currency))
+    )
+    await page.waitForSelector('li.a-last')
+    await page.click('li.a-last')
+  }
+
+  const cleanPrices = filteredPrices.map(function (x) {
```
Comment (maxigimenez) [inline blog/snippets/puppeteer/scraping-example-expenses.js:76]: nit. same arrow function

File: blog/posts/scraping-example-expenses.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+::: tab Puppeteer 
+<<< @/blog/snippets/puppeteer/scraping-example-expenses.js
+:::
+::: tab Playwright
+<<< @/blog/snippets/playwright/scraping-example-expenses.js
+:::
+::::
+
+::: warning
+This example is only intended for learning purposes. Always make sure the website you are planning to scrape allows such behaviour.
```
Comment (maxigimenez) [inline blog/posts/scraping-example-expenses.md:31]: Not sure if worth it, but in my case, I use 2FA for amazon, should we put a warning as well?

**Review by maxigimenez — APPROVED** [review]
LGTM 👍 , just a few nit and one question

File: blog/posts/scraping-example-expenses.md (hunk 4 above)
Comment (ragog) [reply blog/posts/scraping-example-expenses.md:31]: Totally right. I will add it here; 2FA also merits its own section in the linked page in the warning below ("Challenging automation flows")

**Review by tnolet — CHANGES_REQUESTED** [review]

**Pull request merged**